.github/workflows/k8s-related-components-e2e-collector-values.yml

```diff
@@ -0,0 +1,416 @@
+# Default values for opentelemetry-collector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
```

Comment:
Please remove everything from this file expect for the fields that are overridden